samples/msgext-ai-sentiment-analysis/nodejs/Readme.md

```diff
@@ -77,7 +77,7 @@ the Teams service needs to call into the bot.
 
    In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/bot-service/bot-service-quickstart-registration).
     - For bot handle, make up a name.
-    - Select "Use existing app registration" (Create the app registration in Microsoft Entra ID beforehand.)
+    - Select "Use existing app registration" (Create the app registration in Azure Active Directory beforehand.)
```

Comment:
Azure Active Directory to Microsoft Entra ID?

Reply:
changed